### PR TITLE
Remove debug commands, add mblookup command

### PR DESCRIPTION
**Pull request by RecursiveForest**
the `debug` commands were all variously useless internals-exposing tools that were not of significant debugging utility with the exception of `whipper debug musicbrainzngs`, which I have elevated to the top-level command `whipper mblookup`.

### CODE REVIEW

File: whipper/common/mbngs.py [hunk 1]
```diff
@@ -21,9 +21,10 @@
 """
 Handles communication with the MusicBrainz server using NGS.
 """
-
 import urllib2
 
+import whipper
+
 import logging
```
Comment (Freso) [inline whipper/common/mbngs.py]: I would probably do these lines like:
```Python
[…]
"""

import logging
import urllib2

import whipper

logger = …
```

(As per https://www.python.org/dev/peps/pep-0008/#imports)

File: whipper/command/mblookup.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+        metadatas = musicbrainz(discId)
+
+        print '%d releases' % len(metadatas)
+        for i, md in enumerate(metadatas):
+            print '- Release %d:' % (i + 1, )
+            print '    Artist: %s' % md.artist.encode('utf-8')
+            print '    Title:  %s' % md.title.encode('utf-8')
+            print '    Type:   %s' % md.releaseType.encode('utf-8')  # noqa: E501
+            print '    URL: %s' % md.url
```
Comment (Freso) [inline whipper/command/mblookup.py]: The other outputs has the `%s` indented to align with each other, so this one will look out of place when printed. (This seems to be a carry-over from the old code though, so should probably be fixed in its own commit (if not PR).)

File: whipper/command/main.py [hunk 3]
```diff
@@ -81,10 +77,10 @@ class Whipper(BaseCommand):
     subcommands = {
         'accurip': accurip.AccuRip,
         'cd':      cd.CD,
-        'debug':   debug.Debug,
         'drive':   drive.Drive,
         'offset':  offset.Offset,
-        'image':   image.Image
+        'image':   image.Image,
+        'mblookup': mblookup.MBLookup
```
Comment (Freso) [inline whipper/command/main.py]: This lines doesn't align with the rest of the dict. They should probably all have another space added, or all made to only have one space total. (Of course, if you change `mblookup` to `lookup` then it should be fine without altering all the other lines.)

File: whipper/common/mbngs.py (hunk 1 above)
Comment (RecursiveForest) [reply whipper/common/mbngs.py]: It's standard to import logging on its own line right above where you define `logger`, so I'd like to keep it where it is.

File: whipper/command/mblookup.py (hunk 2 above)
Comment (RecursiveForest) [reply whipper/command/mblookup.py]: good catch-- should be fixed in this PR.

File: whipper/command/main.py (hunk 3 above)
Comment (RecursiveForest) [reply whipper/command/main.py]: Yeah I know-- I should un-align the rest of them for consistency.

File: whipper/common/mbngs.py (hunk 1 above)
Comment (MerlijnWajer) [reply whipper/common/mbngs.py]: Right, but usually you first import system libs, then third party libs, then your own. I think that is what @Freso meant?

**Comment by MerlijnWajer** [comment]
Changes look good - didn't test the code. ACK from me.

**Comment by MerlijnWajer** [comment]
mblookup for filename, mblookup for cmd seems sensible to me.

File: whipper/common/mbngs.py (hunk 1 above)
Comment (RecursiveForest) [reply whipper/common/mbngs.py]: It is! But I'm saying that despite pep8 it's common practice to import logging on its own line right above logger, out of standard order. As far as I can tell I'm otherwise following the standard import order. If someone feels really strongly about this I'm open to changing it, but this is the style whipper is currently written in and if we decide to change the import order for logging we should do it wholesale in another PR. (I'm fine discussing it here, however.)

File: whipper/command/mblookup.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        self.parser.add_argument(
+            'mbdiscid', action='store', help="MB disc id to look up"
+        )
+
+    def do(self):
+        try:
+            discId = unicode(self.options.mbdiscid)
+        except IndexError:
+            print 'Please specify a MusicBrainz disc id.'
+            return 3
```
Comment (JoeLametta) [inline whipper/command/mblookup.py]: The return value of 3 has no specific meaning, right? (just returning non-zero)

**Comment by JoeLametta** [comment]
Will test it today: I'd like to merge this PR then tag a new release (`v0.7.0`).

File: whipper/command/mblookup.py (hunk 4 above)
Comment (RecursiveForest) [reply whipper/command/mblookup.py]: It may be semantic, but I haven't checked if it means anything by a particular standard. It's left over from morituri.
